frontend/apps/web/components/site-header/AccountStatusHandler.tsx

```diff
@@ -26,6 +28,14 @@ export function AccountStatusHandler(props: Props) {
 
   return (
     <div className="flex flex-row items-center gap-2">
+      {!systemAppConfig.isNeosyncCloud && (
+        <TrialCountdown
```

Comment:
Seems like this will show for all accounts regardless of their account type or status? Based on what I see in the component the default state is "expired", so even if you are a valid account it will show this but in expired state? Unless I am reading that wrong.